### PR TITLE
Change perform Interaction send INVALID _ID instead REJECTED

### PR DESCRIPTION
**Pull request by SKobziev**
Fixes [#1884](https://github.com/SmartDeviceLink/sdl_core/issues/1884)

This PR is **ready** for review.

### Risk
This PR makes **minor** API changes.

### Testing Plan
ATF script provided at [PR](https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2054)

### Summary
In case of processing DeleteInteractionChoiceSet HMI Perform Interaction send INVALID _ID instead REJECTED.
This commit corrected this behavior in PerformInteractionRequest::CheckChoiceIDFromRequest.

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by SKobziev** [comment]
@AByzhynar @LitvinenkoIra  @mked-luxoft  @ZhdanovP  Could you please review my changes?

File: src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_interaction_request.cc [hunk 1]
```diff
@@ -153,14 +153,16 @@ void PerformInteractionRequest::Run() {
...
-  if (!CheckChoiceIDFromRequest(
-          app,
-          choice_set_id_list_length,
-          msg_params[strings::interaction_choice_set_id_list])) {
+  const auto result = CheckChoiceIDFromRequest(
+      app,
+      choice_set_id_list_length,
+      msg_params[strings::interaction_choice_set_id_list]);
+  if (mobile_apis::Result::SUCCESS != result) {
+    LOG4CXX_DEBUG(logger_, "HERE: " << result);
```
Comment (mked-luxoft) [inline src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_interaction_request.cc]: @SKobziev , is this log still needed here? in case it is, please rephrase it to be more verbose

Comment (SKobziev) [reply src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_interaction_request.cc]: This comment was removed because it was here only for a test. Could you please continue review?

**Comment by SKobziev** [comment]
look at https://github.com/smartdevicelink/sdl_core/pull/2772

**Comment by LuxoftAKutsan** [comment]
@SKobziev  what is the reason for recreating PR?
